app/decorators/db_activity_decorator.rb

```diff
@@ -0,0 +1,14 @@
+class DbActivityDecorator < ApplicationDecorator
+  def resource_diff_table
```

Comment:
Assignment Branch Condition size for resource_diff_table is too high. [18.25/15]